### PR TITLE
topology: add missing tokens in SSP_CONFIG_DATA

### PR DESCRIPTION
**Pull request by juimonen**
Add all same tokens to SSP_CONFIG_DATA that we have in
SSP_MULTI_CONFIG_DATA.

Signed-off-by: Jaska Uimonen <jaska.uimonen@intel.com>

### CODE REVIEW

File: tools/topology/platform/common/ssp.m4 [hunk 1]
```diff
@@ -39,9 +39,14 @@ define(`SSP_CONFIG_DATA',
 `		SOF_TKN_INTEL_SSP_SAMPLE_BITS'	STR($3)
 `		SOF_TKN_INTEL_SSP_QUIRKS'	`ifelse(`$5', `', "0", STR($5))'
 `		SOF_TKN_INTEL_SSP_BCLK_DELAY'	`ifelse(`$6', `', "0", STR($6))'
+`		SOF_TKN_INTEL_SSP_CLKS_CONTROL' `ifelse(`$7', `', "0", STR($7))'
```
Comment (ranj063) [inline tools/topology/platform/common/ssp.m4:42]: missed updating the macro definition comment above?

**Comment by lgirdwood** [comment]
@juimonen does the kernel need a similar fix ? or does this fix mean the kernel needs no change ?

**Comment by juimonen** [comment]
> @juimonen does the kernel need a similar fix ? or does this fix mean the kernel needs no change ?

these are already in kernel. We are using them already in multi ssp case. So not sure why they we're not used in "single" ssp case. maybe there was no need...?

File: tools/topology/platform/common/ssp.m4 (hunk 1 above)
Comment (juimonen) [reply tools/topology/platform/common/ssp.m4:43]: ok, added

**Review by lgirdwood — APPROVED** [review]
@juimonen is this also for rrc3 (I would assume it is) ?

**Comment by juimonen** [comment]
@lgirdwood, original author of the bug @plbossart was away so not sure is this enough for him... also not totally sure could we have some impact for existing single ssp platforms? All the test are green and my own take is that this _should_ not have any effect -> in my opinion this could just go in to rc3

**Review by plbossart — APPROVED** [review]
That's what I suggested indeed, and if we pass CLKS_CONTROL we have a mechanism to provide the early_ssp_start capability that we need. @bardliao FYI.

**Pull request merged**